release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2775,6 +2775,53 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.15.42
+[id="ocp-4-15-42_{context}"]
+=== RHSA-2024:11562 - {product-title} 4.15.42 bug fix and security update
+
+Issued: 02 January 2025
+
+{product-title} release 4.15.42, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:11562[RHSA-2024:11562] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:11565[RHBA-2024:11565] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.42 --pullspecs
+----
+
+[id="ocp-4-15-42-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when the webhook token authenticator was enabled and had the authorization type set to `None`, the {product-title} web console would consistently crash. With this release, a bug fix ensures that this configuration does not cause the {product-title} web console to crash. (link:https://issues.redhat.com/browse/OCPBUGS-46482[*OCPBUGS-46482*])
+
+* Previously, when you attempted to use {olm-first} to upgrade an Operator, the upgrade was blocked and an `error validating existing CRs against new CRD's schema` message was generated. An issue existed with {olm}, whereby {olm} erroneously identified incompatibility issues validating existing custom resources (CRs) against the new Operator version's custom resource definitions (CRDs). With this release, the validation is corrected so that Operator upgrades are no longer blocked. (link:https://issues.redhat.com/browse/OCPBUGS-46479[*OCPBUGS-46479*])
+
+* Previously, the images for custom OS layering were not present when the OS was on {rhcos-short} 4.15, preventing some customers from upgrading from {rhcos-short} 4.15 to {rhcos-short} 4.16. This release adds ACR and GCR image credential provider RPMs to {rhcos-short} 4.15. (link:https://issues.redhat.com/browse/OCPBUGS-46063[*OCPBUGS-46063*])
+
+* Previously, it was impossible to configure your Amazon Web Services DHCP option set with a custom domain name containing a trailing period, as trailing periods were not allowed in a Kubernetes object name. With this release, trailing periods are allowed in a domain name in a DHCP option set. (link:https://issues.redhat.com/browse/OCPBUGS-46034[*OCPBUGS-46034*])
```

Comment:
I'm not 100% sure, but is "trailing period" something like `domainname.`? Might rephrase as:
```suggestion
* Previously, you could not configure your Amazon Web Services DHCP option set with a custom domain name containing a period (`.`) as the final character, as trailing periods were not allowed in a Kubernetes object name. With this release, trailing periods are allowed in a domain name in a DHCP option set. (link:https://issues.redhat.com/browse/OCPBUGS-46034[*OCPBUGS-46034*])
```